engine/evarutil.ml

```diff
@@ -740,6 +740,13 @@ let undefined_evars_of_named_context evd nc =
     nc
     ~init:Evar.Set.empty
 
+let undefined_evars_of_econstr_named_context evd nc =
+  Context.Named.fold_outside
```

Comment:
If only we were using standard names, eg `fold_right` here one could ask why not using `fold_left` given that `union` is commutative and `fold_left` is more stack efficient.